token-minter/main.go

```diff
@@ -65,106 +65,113 @@ func main() {
 		os.Exit(1) // second signal. Exit directly.
 	}()
 
-	expirationTimestamp := mintToken(ctx, opts)
-
 	if opts.oneshot {
-		return
+		_, err := mintToken(ctx, opts)
+		if err != nil {
+			log.Fatalln(err)
+		}
+		os.Exit(0)
 	}
 
+	renewDuration := 1 * time.Second
```

Comment:
why set to 1s?  why not just `var renewDuration time.Duration` and move it inside the `for`?

Reply:
I guess it could be the zero value (`int64`) and it would work the same

Reply:
Fixed